docs/reference/licenses.nuget.org.md

```diff
@@ -0,0 +1,59 @@
+# licenses.nuget.org
+
+## Rationale
+
+With introduction of the [license expressions](nuspec.md#license) a requirement to have a reliable service that would
```

Comment:
Move emerged to after requirement. It makes the sentence easier to understand

`requirement *emerged* to have`